elm-package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.0.0",
+    "summary": "helpful summary of your project, less than 80 characters",
+    "repository": "https://github.com/user/project.git",
+    "license": "BSD3",
+    "source-directories": [
+        "."
```

Comment:
I'm going to add one more stage. We can rename the directories to be One,
Two, Three, etc, or add elm-package.json.
On Sat, Apr 16, 2016 at 5:17 PM Paul Coyle <notifications@github.com> wrote:

> In elm-package.json
> <https://github.com/paulcoyle/yycjs-elm/pull/1#discussion_r59975623>:
>
> > @@ -0,0 +1,16 @@
> > +{
> > +    "version": "1.0.0",
> > +    "summary": "helpful summary of your project, less than 80 characters",
> > +    "repository": "https://github.com/user/project.git",
> > +    "license": "BSD3",
> > +    "source-directories": [
> > +        "."
>
> Forgot to mention that this is going to be problematic for importing
> modules within the project (even if set to src). Since the stages are
> numbers we won't be able to import as 7.Menu for example (leading with a
> number is syntactically incorrect). We may need to put an elm-package.json
> in each stage.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/paulcoyle/yycjs-elm/pull/1/files/ecc84537dcfb3e36c3966ab42c90529ec4b06a52#r59975623>
>